src/content/learn/editor-setup.md

```diff
@@ -1,62 +1,62 @@
 ---
-title: Editor Setup
+title: એડિટર સેટઅપ
 ---
 
 <Intro>
 
-A properly configured editor can make code clearer to read and faster to write. It can even help you catch bugs as you write them! If this is your first time setting up an editor or you're looking to tune up your current editor, we have a few recommendations.
+યોગ્ય રીતે ગોઠવેલ એડિટર કોડને વાંચવા માટે સ્પષ્ટ અને લખવા માટે ઝડપી બનાવી શકે છે. તે તમને ભૂલો લખતા જ તેને પકડવામાં પણ મદદ કરી શકે છે! જો આ તમે પ્રથમ વખત એડિટર સેટઅપ કરી રહ્યાં છો અથવા તમે તમારા વર્તમાન એડિટર ને ટ્યુન કરવા માટે શોધી રહ્યાં છો, તો અમારી પાસે કેટલીક ભલામણો છે.
 
 </Intro>
 
 <YouWillLearn>
 
-* What the most popular editors are
-* How to format your code automatically
+* સૌથી વધુ લોકપ્રિય એડિટરો કયા છે
+* તમારો કોડ આપમેળે કેવી રીતે ફોર્મેટ કરવો
 
 </YouWillLearn>
 
-## Your editor {/*your-editor*/}
+## તમારુ એડિટર {/*your-editor*/}
 
-[VS Code](https://code.visualstudio.com/) is one of the most popular editors in use today. It has a large marketplace of extensions and integrates well with popular services like GitHub. Most of the features listed below can be added to VS Code as extensions as well, making it highly configurable!
+[VS Code](https://code.visualstudio.com/) એ આજે ઉપયોગમાં લેવાતા સૌથી લોકપ્રિય એડિટરો પૈકીનું એક છે. તેની પાસે એક્સ્ટેંશનનું વિશાળ બજાર છે અને GitHub જેવી લોકપ્રિય સેવાઓ સાથે સારી રીતે સંકલિત છે. નીચે સૂચિબદ્ધ મોટાભાગની સુવિધાઓ VS Code માં એક્સ્ટેંશન તરીકે પણ ઉમેરી શકાય છે, જે તેને અત્યંત રૂપરેખાંકિત બનાવે છે!
```

Comment:
Also here રૂપરેખાંકિત its not daily life Gujarati word. What about કોન્ફિગરેબલ.

Reply:
Yes, that makes good meaning